### PR TITLE
switch Digital Mars copyright to D Language Foundation

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: std/bitmanip.d [hunk 1]
```diff
@@ -34,7 +34,7 @@ $(TR $(TD Tagging) $(TD
 ))
 )
 
-Copyright: Copyright Digital Mars 2007 - 2011.
+Copyright: Copyright The D Language Foundation" 2007 - 2011.
```
Comment (n8sh) [inline std/bitmanip.d]: Does the `"` do something in ddoc or should it be removed?

Comment (ibuclaw) [reply std/bitmanip.d]: Looks like a bad find/replace to me.

Comment (BBasile) [reply std/bitmanip.d]: The "Copyright" section doesn't even appear in the generated doc. But at the bottom the footer is already correct.

Comment (BBasile) [reply std/bitmanip.d]: Shortcut for new [bitmanip doc](http://dtest.dlang.io/artifact/website-748d1fb3b3acc80ef7bc5c69516165dc709b330b-d4c7dd79f99f3fa706c7f547db77e590/web/phobos-prerelease/std_bitmanip.html). No copyright at all at the top.

**Pull request merged**